src/services/user.go

```
@@ -66,6 +66,34 @@ func (service userService) Info(ctx context.Context, request domainUser.InfoRequ
+		return response, err
+	}
+	
+	resp, err := service.WaCli.IsOnWhatsApp([]string{request.Phone})
+	if err != nil {
+		return response, err
+	}
+
+	uc := new(domainUser.UserCollection)
+	for _, item := range resp {
+		if item.VerifiedName != nil {
+			var msg = domainUser.CheckResponseData{Query: item.Query, IsInWhatsapp: item.IsIn, JID: fmt.Sprintf("%s", item.JID), VerifiedName: item.VerifiedName.Details.GetVerifiedName()}
```

golangci-lint (1.62.2)

S1025: should use String() instead of fmt.Sprintf

(gosimple)